lepidopter-fh/configure.sh

```diff
@@ -3,3 +3,10 @@ set -ex
 
 # Set locatime to UTC
 cp /usr/share/zoneinfo/UTC /etc/localtime
+
+# Checkout lepidopter-update repo
+git clone https://github.com/OpenObservatory/lepidopter-update \
```

Comment:
After talking with @hellais we decided that currently it's better if we include the `updater.py` script in lepidopter git repo rather than checking out the git repo of lepidopter-update